### PR TITLE
Close #1 - Create components for Status Nav, Tab, Tool and Snack Bars

### LINKED ISSUE #1: Create components for Status Nav, Tab, Tool and Snack Bars
We are in the process of moving some of our design work from Figma to react-storybooks so that it is:

1. Easier for developers to work quickly and iterate
2. Easier for designers and developers to work in tandem and provide functional feedback on each other's requirements and proposals.
3. We can continue moving toward a more standard library of components that can be used across Status, as well as by the wider community of DApp developers.

The first components we'd like built are the Navigation, Tab, Tool, and Snack bars [here](https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=0%3A1).

There will be a bounty for the issue after a little more discussion/clarification.

### PR DESCRIPTION
**Pull request by millnitzluan**
Created components to close PR #1.
This is work in progress

### CODE REVIEW

**Comment by andytudhope** [comment]
@bgits wdyt?

**Comment by millnitzluan** [comment]
@andytudhope , I think that styles needs to be configured in the material-ui.

https://material-ui.com/style/color/

**Comment by bgits** [comment]
The actual stories need to have knobs, see `Button > Select Type`  as an example.

**Comment by bgits** [comment]
The navigation bar has a shadow, the design is flat.

**Comment by millnitzluan** [comment]
@bgits The design is with the same with sketch, the difference is because the Navigation is with width 100% of storybook container, in small screen it's look fit. I'm refactoring the components, the chat bar and and group chat I'll create other component for it. Sorry for the delay, it's because this week I'm to busy.

**Comment by millnitzluan** [comment]
@bgits and @andytudhope can you check again?

File: src/components/Chatbar.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    marginLeft: -12
+  },
+  badge: {
+    top: 1,
+    right: -15
+  },
+};
+
+function Chatbar({
+  classes, prymary, name, avatarUrl, description, style, handlePreviousClick, menu, handleMenuClick, unreadMessages, ...props
```
Comment (bgits) [inline src/components/Chatbar.jsx]: `prymary` should be `primary`

File: src/components/Navigation.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            {title}
+          </Typography>
+          { detailsTypography() }
+        </div>
+        { menuIcon() }
+      </Toolbar>
+    </AppBar>
+  )
+}
+
```
Comment (bgits) [inline src/components/Navigation.jsx]: can you add propTypes for this component here?

File: src/components/Navigation.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    flexGrow: 1,
+    textDecoration: 'none'
+  },
+  menuButton: {
+    marginLeft: -12,
+    marginRight: 20
+  },
+};
+
+function Navigation({
```
Comment (bgits) [inline src/components/Navigation.jsx]: This component is missing badges for indicating a number and image as shown for the navigation components labeled 'Chat' in Figma.

File: src/components/Tabbar.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    this.setState({ value });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { value } = this.state;
+
+    return (
+      <Paper className={classes.root}>
+        <Tabs
```
Comment (bgits) [inline src/components/Tabbar.jsx]: I think the actual tabs need to be `space-evenly` or `space-between` to be aligned with the design in Figma

**Comment by bgits** [comment]
Added some comments, I think we are very close to wrapping this up. @millnitzluan you think it makes sense to have ChatBar be a separate component from NavigationBar? @andytudhope Thoughts?

**Comment by bgits** [comment]
> What do you think about creating a issue to configure material-ui theme, so this treatment of color don't need to be done in the component, I think that the material-ui can do it for us.

I think it's a good point. We will consider it more closely when we get color direction from our designers.

**Comment by bgits** [comment]
@millnitzluan @andytudhope There are just the 3 comments I left on the actual code portions themselves, if we can just have those addressed and rebase the PR against master I think we can wrap this up.

File: src/components/Tabbar.jsx (hunk 4 above)
Comment (millnitzluan) [reply src/components/Tabbar.jsx]: Done

File: src/components/Navigation.jsx (hunk 2 above)
Comment (millnitzluan) [reply src/components/Navigation.jsx]: Done

**Comment by millnitzluan** [comment]
@bgits Done!

**Comment by bgits** [comment]
LGTM. @millnitzluan If you can just rebase from our master, as we can not merge this.

**Comment by andytudhope** [comment]
Cool, the bounty has been paid 👍

**Pull request merged**